packages/rune-games-cli/src/flows/Create.tsx

```diff
@@ -175,10 +179,14 @@ export function Create({ args }: { args: string[] }) {
         )
       )}
       {step === Steps.Done && (
-        <Text>{`\nTo start the project, run:
+        <Text>{`
+To start the project, run:
```

Comment:
```suggestion
\nTo start the project, run:
```

IMO it's nice to keep the line break for clarity

Reply:
Not sure I understand. We are inside of template literal so new line in code will render new line in terminal, i.e. this code did not make any changes regarding rendering.

Reply:
Ah, makes sense 👍 I've updated my other suggestion to also use a new line instead of a line break.